news/2024/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area.md

```diff
@@ -0,0 +1,307 @@
+
+![](/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/euphoria.jpg)
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+What goals in osu! do you have that you would like to achieve in the future?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+I still have some dream scores I'd like to do, like finishing all my FC's on the Because Maybe's, maybe a cheeky HDHR FC on Sound Chimera, Euphoria, and Firelight... amongst some other goals. Competitively, I'd still like an international 1v1 badge, and to continue trying and supporting new and innovative ideas, like the recent Lazer Grand Arena tournament!
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/18152711" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-MegaMix.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-lvd" href="https://osu.ppy.sh/users/18152711">MegaMix</a></p>
+
+And as a finishing touch, do you have anything else that you'd like to mention?
+
+<a class="avatar news-chat-quote__avatar" href="https://osu.ppy.sh/users/3717598" style="background-image: url('/wiki/shared/news/2024-09-16-the-followpoint-xootynator-the-ceo-of-full-area/avatar-xootynator.jpg')"></a>
+
+<p class="news-chat-quote__username"><a class="news-chat-quote__colour-no-group" href="https://osu.ppy.sh/users/3717598">xootynator</a></p>
+
+I hope to continue doing my best with osu! and with content creation, and I hope I can contribute more to the game as time goes on. One of my favourite contributions has been the tournament I co-host with Fulserish, dubbed "Battle of the Box", which encouraged a lot of artists in the community to contribute to tournaments, which I think had a domino effect where many more tournaments are featuring custom artwork for custom maps! If you would like to get involved in that, please consider joining the [osu! Artist Hub](https://discord.gg/KVPDmKAkjV) Discord, where there are a lot of like-minded individuals who want to be better artists and continue contributing to the game.
```

Comment:
is there something that can be linked regarding battle of the box? like a forum thread or something

Reply:
```suggestion
I hope to continue doing my best with osu! and with content creation, and I hope I can contribute more to the game as time goes on. One of my favourite contributions has been the tournament I co-host with [Fulserish](https://osu.ppy.sh/users/14252162), dubbed "Battle of the Box", which encouraged a lot of artists in the community to contribute to tournaments, which I think had a domino effect where many more tournaments are featuring custom artwork for custom maps! If you would like to get involved in that, please consider joining the [osu! Artist Hub](https://discord.gg/KVPDmKAkjV) Discord, where there are a lot of like-minded individuals who want to be better artists and continue contributing to the game.
```

Reply:
Will try to find the link rq